scripts/build.js

```diff
@@ -100,6 +100,14 @@ async function generateCalendar() {
       fs.mkdirSync(outputDir, { recursive: true });
     }
 
+    // Create a new JSON file with an array of months that have events
+    const eventsFileOutputPath = path.join(outputDir, "events.js");
+    const jsContent = `
+    const eventsData = ${JSON.stringify(Object.keys(events), null, 2)};
+    export default eventsData;`;
```

Comment:
Nice!

Let's simplify it further.

1. I don't think we need to bind it to a variable first. We can export directly.
2. We don't necessarily need to extra arguments to `JSON.stringify`
3. Let's return a `Set` so that we can query more efficiently later on

```suggestion
    const jsContent = `export default new Set(${JSON.stringify(Object.keys(events))});`;
```